### PR TITLE
added cleanup for the temporary test package and documentation update

### LINKED ISSUE #292: /tests/test_packages/functional/package_test.md - outdated
@gyorb , maybe we could remove this doc, then extend the user guide with a how-to-run-automated-tests paragraph (tell the users to run /run_tests.sh).

### PR DESCRIPTION
**Pull request by gyorb**
- added cleanup for the temporary test package
- test documentation update

resolves #292

### CODE REVIEW

File: tests/package_test.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+~~~~~~
+
+## More detailed test running
+~~~~~~{.sh}
+# setup some required environment variables to run the tests
+# create temporary directories
+source tests/test_env_setup.sh
+
+# build the CodeChecker package
+build_package.py -o $TEST_CODECHECKER_PACKAGE_DIR -v
```
Comment (bruntib) [inline tests/package_test.md]: Help message for `-v` flag in `build_package.py` is missing.

File: tests/package_test.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# setup the environment to use SQLite database for testing
```
Comment (bruntib) [inline tests/package_test.md]: In this step we require the user to set `TEST_USE_POSTGRESQL` and `CODECHECKER_DB_DRIVER` environment variables. The former one is quite descriptive, but the meaning of the latter one is not clear. When we provide commands to the user i.e. commands are not run by scripts, then those should be described.

File: tests/package_test.md (hunk 1 above)
Comment (gyorb) [inline tests/package_test.md]: help added

File: tests/package_test.md (hunk 2 above)
Comment (gyorb) [inline tests/package_test.md]: I extended the documentation with PosgtreSQL setup for testing

**Pull request merged**